plugins/services/src/js/utils/__tests__/MarathonErrorUtil-test.js

```diff
@@ -185,21 +185,21 @@ describe("MarathonErrorUtil", function() {
       ]);
     });
 
-    it("should handle a message value of empty object as empty", function() {
+    it("handles a message value of empty object as empty", function() {
       // TODO (JIRA DCOS_OSS-653): We should include these with a generic error
```

Comment:
🚨 Comment (not yours)

Reply:
Not sure why you comment on lines like this one.

Reply:
because it was late 😆 feel free to ignore them